recipe/meta.yml

```diff
@@ -15,7 +15,7 @@ requirements:
   run:
     - conda
     - python >=3.7
-    - art
+    - pyfiglet
```

Comment:
I think this won't work since `pyfiglet` isn't in defaults and the conda-build recipe won't actually build. I wonder if there is any other ASCII art library that is already in defaults?